From 3b58acf8a4d754e5e45c0723c71525ae79ad7bda Mon Sep 17 00:00:00 2001
From: Thu B Nguyen <tbnguyen@amperecomputing.com>
Date: Mon, 12 Oct 2020 21:47:10 +0700
Subject: [PATCH 3/9] dt-bindings: i2c: ampere, smpro: Add binding for Ampere
 SMpro hwmon driver

Add device tree binding document for Ampere SMpro hwmon driver

Signed-off-by: Thu B Nguyen <tbnguyen@amperecomputing.com>
---
 .../bindings/i2c/ampere,ac01-smpro.yaml       | 31 +++++++++++++++++++
 MAINTAINERS                                   |  7 +++++
 2 files changed, 38 insertions(+)
 create mode 100644 Documentation/devicetree/bindings/i2c/ampere,ac01-smpro.yaml

diff --git a/Documentation/devicetree/bindings/i2c/ampere,ac01-smpro.yaml b/Documentation/devicetree/bindings/i2c/ampere,ac01-smpro.yaml
new file mode 100644
index 000000000000..b2e0f4b3d4a3
--- /dev/null
+++ b/Documentation/devicetree/bindings/i2c/ampere,ac01-smpro.yaml
@@ -0,0 +1,31 @@
+# SPDX-License-Identifier: (GPL-2.0 OR BSD-2-Clause)
+%YAML 1.2
+---
+$id: "http://devicetree.org/schemas/i2c/ampere,ac01-smpro.yaml#"
+$schema: "http://devicetree.org/meta-schemas/core.yaml#"
+
+title: Ampere SMpro controller Device Tree Bindings
+
+maintainers:
+  - thu@os.amperecomputing.com
+
+allOf:
+  - $ref: /schemas/i2c/i2c-controller.yaml#
+
+properties:
+  compatible:
+    const: ampere,ac01-smpro
+
+  reg:
+    maxItems: 1
+
+required:
+  - compatible
+  - reg
+
+examples:
+  - |
+        smpro@4f {
+                compatible = "ampere,ac01-smpro";
+                reg = <0x4f>;
+        };
diff --git a/MAINTAINERS b/MAINTAINERS
index 5fecb388d073..a422c614308e 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -934,6 +934,13 @@ S:	Supported
 F:	arch/arm64/boot/dts/amd/amd-seattle-xgbe*.dtsi
 F:	drivers/net/ethernet/amd/xgbe/
 
+AMPERE COMPUTING SMpro DRIVER
+M:	Thu Ba Nguyen <thu@os.amperecomputing.com>
+S:	Supported
+F:	Documentation/devicetree/bindings/i2c/ampere,ac01-smpro.yaml
+F:	arch/arm/boot/dts/aspeed-bmc-ampere-mtjade.dts
+F:	drivers/hwmon/smpro-hwmon.c
+
 ANALOG DEVICES INC AD5686 DRIVER
 M:	Michael Hennerich <Michael.Hennerich@analog.com>
 L:	linux-pm@vger.kernel.org
-- 
2.18.4

